src/command/verify/verify.c

```diff
@@ -1632,6 +1632,63 @@ verifyProcess(unsigned int *errorTotal)
     FUNCTION_LOG_RETURN(STRING, result);
 }
 
+/***********************************************************************************************************************************
+Format the text result
+***********************************************************************************************************************************/
+static String *
+verifyOutputText(const String *verifyresult, const unsigned int *errorTotal)
+{
+    FUNCTION_LOG_BEGIN(logLevelDebug);
+        FUNCTION_TEST_PARAM(STRING, verifyresult);                  // Result string from verifyProcess()
+    FUNCTION_LOG_END();
+
+    String *result = NULL;
+
+    MEM_CONTEXT_TEMP_BEGIN()
+    {
+        String *resultStr = strNew();
+        String *status = strNew();
+
+        if (*errorTotal == 0)
+            strCatFmt(status,"%s%s",  strZ(strNewZ("All archives and backups passed verification.")), LF_Z);
+        else
+        {
+            strCatFmt(
+                status,
+                "%s%s",
+                strZ(strNewZ("One or more archive or backup contains errors. Invoke verify with verbose option for more details.")),
+                LF_Z);
+        }
+
+        if (cfgOptionStrId(cfgOptOutput) == CFGOPTVAL_OUTPUT_TEXT)
+        {
+            // Output verbose response if requested
+            if (cfgOptionBool(cfgOptVerbose))
+            {
+                strCat(resultStr, strNewFmt("Stanza: %s", strZ(cfgOptionStr(cfgOptStanza))));
+                strCat(resultStr, strNewFmt("%sStatus: %s", LF_Z, strZ(status)));
+                strCatFmt(
+                    resultStr, "%s%s", strZ(strSub(verifyresult, (unsigned int)strChr(verifyresult, ':') + 1)), LF_Z);
+            }
+            else
+            {
+                strCatFmt(
+                    resultStr, "%s%s%s", strZ(strNewFmt("Stanza: %s", strZ(cfgOptionStr(cfgOptStanza)))), LF_Z,
+                    strZ(strNewFmt("Status: %s", strZ(status))));
+            }
+        }
+
+        MEM_CONTEXT_PRIOR_BEGIN()
+        {
+            result = strDup(resultStr) ;
```

Comment:
extra space before `;`?

Reply:
addressed in 1ab4e8eb20873848a78df60be92e2356098786ec